frontend/src/component/common/Notifications/Notifications.tsx

```diff
@@ -0,0 +1,61 @@
+import Settings from '@mui/icons-material/Settings';
+import { Paper, Typography, Box, IconButton } from '@mui/material';
+import { useNotifications } from 'hooks/api/getters/useNotifications/useNotifications';
+
+export const Notifications = () => {
+    const { notifications } = useNotifications({ refreshInterval: 15 });
+
+    console.log(notifications);
```

Comment:
console.log can be remove

Reply:
Yes, I'm still using it. It will be removed in the followup